homeassistant/components/onvif/sensor.py

```diff
@@ -40,26 +56,42 @@ def async_check_entities():
     return True
 
 
-class ONVIFSensor(ONVIFBaseEntity, SensorEntity):
+class ONVIFSensor(ONVIFBaseEntity, RestoreSensor):
```

Comment:
Was it intentional to remove the inheritance from `SensorEntity`?

In `binary_sensor.py`, you kept the inheritance from `BinarySensorEntity`, but here it is dropped - not sure why.

Reply:
`RestoreSensor` already inherits from `SensorEntity`, but `RestoreEntity` does not inherit from `BinarySensorEntity`